### ISSUE TITLE
rootmy.tv on text mode

### ISSUE THREAD

**Issue by brabrablox100** [RootMyTV/RootMyTV.github.io#62]
when i try to go on the website it just becomes text only but i can still click to start or press 5 but theres an error that says "script undefined undefined" how do i fix this

**Comment by DavidBuchanan314** [comment]
Which webOS version are you running? It sounds like it might be something ancient.

**Comment by brabrablox100** [comment]
> Which webOS version are you running? It sounds like it might be something ancient.

1.4.0-2536

**Comment by brabrablox100** [comment]
thats the webos tv version

**Comment by shatallik** [comment]
Friends, Hi! Tell me When rootmytv 3.0?

**Comment by DavidBuchanan314** [comment]
> 1.4.0-2536

That sounds like the firmware version, not the webOS version. You should be able to find it under **Settings > General > About This TV > TV Information > webOS TV Version**

**Comment by GiorgioRen** [comment]
I have the same problem, my tv's webOS version is 05.05.90
The webpage loads correctly but just for a split second, after that it turns to just text (no css)

If I press on 5 it asks for permission but then a prompt appears saying `error: "Script error." undefined undefined`

**Comment by Informatic** [comment]
This is probably the same thing - this is webOS 1.x or 2.x. (05.05.90 is "Software Version" which is a different thing)

**Comment by GiorgioRen** [comment]
Yes you're right, sadly my webOS version is 1.4.0-2436 so no homebrew :(
Thank you!
